scipy/stats/_axis_nan_policy.py

```diff
@@ -259,6 +259,14 @@ def _check_empty_inputs(samples, axis):
                                           inspect.Parameter.KEYWORD_ONLY,
                                           default='propagate')
 
+_standard_note_addition = (
+    """\nBeginning in SciPy 1.9, ``np.matrix`` inputs are converted to
+``np.ndarray``s before the calculation is performed. In this case, the
```

Comment:
Since we are saying this, I would go into saying that `np.matrix` is not recommended anymore by NumPy. Even-though it's arguably our role to educate people on NumPy's current best practices.